### PR TITLE
John conroy/prettier

### PR DESCRIPTION
**Pull request by john-conroy**
Towards #280 . Needed to install node v10.0+ in travis for prettier to work.

### CODE REVIEW

**Comment by john-conroy** [comment]
It's just failing due to the new prettier errors. Chuck do you want to take a look at the changes before I go ahead and reformat everything?

File: context/.prettierrc.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+semi: true
+trailingComma: "all"
+singleQuote: true
+printWidth": 70
```
Comment (mccalluc) [inline context/.prettierrc.yml:4]:
```suggestion
printWidth: 70
```

Comment (mccalluc) [reply context/.prettierrc.yml:4]: Also, would you like this to be higher? Perhaps look at the reformatted code and see if it's doing too much wrapping?

Comment (john-conroy) [reply context/.prettierrc.yml]: Yeah, it was much too drastic. I increased it to 120.

**Comment by john-conroy** [comment]
Adding prettier required some vscode workspace settings to get eslint to work properly. I think it makes sense to include those workspace specific settings in version control.

**Pull request merged**